### ISSUE TITLE
AndroidX links to documentation

### ISSUE THREAD

**Issue by BlastyCZ** [Kotlin/dokka#498]
I'm using androidx libraries in my project, while I was writing a docs I found out that links to methods are not working, it is caused by bad URL obviously. The problem is the last slash between class name and method name.

> Generated link: https://developer.android.com/reference/kotlin/androidx/recyclerview/widget/DiffUtil/ItemCallback.html
> Working link: https://developer.android.com/reference/kotlin/androidx/recyclerview/widget/DiffUtil.ItemCallback.html
> 

My dokka setting:
```
    externalDocumentationLink {
        url = new URL("https://developer.android.com/reference/kotlin/")
        packageListUrl = new URL("https://developer.android.com/reference/androidx/package-list")
    }
```
Is there any workaround or is my setting wrong or did I miss anything else?

**Comment by kamilok1965** [comment]
Thanks for your contribution! 
It's probably a bug, I'll investigate that

**Comment by tynn** [comment]
This was addressed in #221 and #223. Slightly outdated though.

**Comment by kamildoleglo** [comment]
AFAIK fixed in `1.4.10.2`
